webapp/graphql/User.js

```diff
@@ -283,3 +283,12 @@ export const currentUserQuery = gql`
     }
   }
 `
+
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-05-27T11:05:08Z" title="Wednesday, May 27th 2020, 1:05:08 pm +02:00">May 27, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -265,9 +265,11 @@ export const checkSlugAvailableQuery = gql`
 
 export const currentUserQuery = gql`
   ${userFragment}
+  ${userCountsFragment}
```


I think you have forgotten to delete this?

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-07-30T11:40:57Z" title="Thursday, July 30th 2020, 1:40:57 pm +02:00">Jul 30, 2020</time>_

---


ok